src/popup.html

```diff
@@ -83,13 +83,21 @@ <h4 class="i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></h4>
         <!-- Panel: Masks (Default once logged in) -->
         <masks-panel data-panel-id="masks" class="is-hidden fx-relay-panel" id="masks-panel">
           <div class="fx-relay-panel-content">
+            <!-- CSAT Survey -->
+            <div class="fx-relay-csat-survey-link-container is-hidden" role="alert" aria-labelledby="surveyLabel" aria-hidden="true">
+              <span id="surveyLabel" class="i18n-content fx-relay-csat-survey-link" data-i18n-message-id="popupSurveyLink"></span>
```

Comment:
This still needs to be a keyboard-navigable element. When tabbing via keyboard, I can only click the "X" from the popup: 

<img width="407" alt="image" src="https://github.com/mozilla/fx-private-relay-add-on/assets/2692333/7ab7ac3f-47be-45d3-a708-9c2d4a0b05e4">

Should this also be a `<button>` element? (Same question as seen previously)

Reply:
Converted to button